go.mod

```diff
@@ -22,6 +22,5 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kubectl v0.18.10
 	sigs.k8s.io/cli-utils v0.22.1-0.20201117031003-fd39030f0508
-	sigs.k8s.io/kustomize/cmd/config v0.8.5
```

Comment:
Just ensure the version we forked is recorded in `thirdparty/...`

Reply:
Done. Thanks.